models/wrf/tutorial/README.rst

```diff
@@ -236,6 +236,7 @@ bash  ``export BASE_DIR=<path_to_your_working_directory>``
 
        cp $DART_DIR/models/wrf/tutorial/template/namelist.input.meso   $BASE_DIR/template/.
        cp $DART_DIR/models/wrf/tutorial/template/namelist.wps.template $BASE_DIR/template/.
+       cp $DART_DIR/models/wrf/tutorial/template/input.nml.template    $BASE_DIR/template/.
```

Comment:
input.nml.template is a DART namelist. The comment above talks about WRF namelists. It would be good to be clear about namelists for running WRF vs namelist for running DART executables.

Reply:
Latest commit 911dc4e8bee239edfa2851478b7f4d1d7645dbfc, addresses this.

Reply:
Commits 08aba5ea5af893a65c627b2284529d50c8153e2c and 63a5748de6f7fd775ee58bcf76d23131fbe6a97c eliminate the need for this line in WRF Tutorial instructions.